nl/mail/chrome/messenger/filter.properties

```diff
@@ -53,19 +53,19 @@ filterAction17=gelabeld
 #entities below were automatically added by l10nmerge
 
 
-bodyFilterWarning=Creating a Body filter will automatically mark the Inbox for Offline use in Folder Properties. For this filter to operate on other folders, you must manually set those folders for Offline use in Folder Properties.
-bodyFilterWarningCheck=Don't show this warning again
-systemSound=System Sound
-soundFilePickerTitle=Choose Sound
-installScriptCheck=Yes, please install it
-noScriptSelected=Select an AppleScript
+bodyFilterWarning=Het aanmaken van een Body filter markeert automatisch de Inbox voor Offline gebruik in de Mapeigenschappen. Om dit filter toe te passen op andere mappen, moet u deze mappen handmatig op Offline gebruik zetten in de Mapeigenschappen.
```

Comment:
CORRECTION NEEDED, it should be: ''Het aanmaken van een Body filter markeert automatisch de Inbox voor Offline gebruik in de Mapeigenschappen. Om deze filter toe te passen op andere mappen, moet u deze mappen handmatig op Offline gebruik zetten in de Mapeigenschappen.''

- ''dit filter'' vs. ''deze filter''.